<launch>

  <arg name="n" default="0"/>

  <param name="subjectNumber" value="$(arg n)" />
  <param name="userRadius" value="0.007" />

  <node pkg="wam_impedance_experiment" name="daq" type="daq"/>
  <!-- Removed so the cisst node can be run separately because it needs to be restarted
       periodically due to a memory leak.
  <node pkg="wam_impedance_experiment" name="stubs_control" type="two_stubs_control"/>
  -->
  <node pkg="wam_impedance_experiment" name="logger" type="logger"/>
  <node pkg="wam_impedance_experiment" name="projectiles" type="projectiles"/>
  <!-- Removed so the graphics can be run on a different machine.
  <node pkg="wam_impedance_experiment" name="graphics" type="graphics"/>
  -->
  <node pkg="wam_impedance_experiment" name="experiment_manager" type="experiment_manager"/>

</launch>
